src/chainparamsbase.cpp

```diff
@@ -33,17 +25,21 @@ const CBaseChainParams& BaseParams()
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain)
 {
     if (chain == CBaseChainParams::MAIN)
-        return MakeUnique<CBaseChainParams>("", 8332);
+        return MakeUnique<CBaseChainParams>(chain, "", 8332);
     else if (chain == CBaseChainParams::TESTNET)
-        return MakeUnique<CBaseChainParams>("testnet3", 18332);
+        return MakeUnique<CBaseChainParams>(chain, "testnet3", 18332);
     else if (chain == CBaseChainParams::REGTEST)
-        return MakeUnique<CBaseChainParams>("regtest", 18443);
+        return MakeUnique<CBaseChainParams>(chain, "regtest", 18443);
     else
         throw std::runtime_error(strprintf("%s: Unknown chain %s.", __func__, chain));
 }
 
+bool BaseParamsSelected()
+{
+    return globalChainBaseParams.get() != nullptr;
```

Comment:
Small nit:  Smart pointers (`std::unique_ptr` in this case) allow direct comparison against `nullptr`.  So there's no need to call `get`.

Reply:
Fixed 👍